field/TinyLang/Field/Raw/Core.hs

```diff
@@ -35,6 +35,7 @@ data Expr v f
     | EAppUnOp   UnOp            (Expr v f)
     | EStatement (Statement v f) (Expr v f)
     | EIf        (Expr v f)      (Expr v f) (Expr v f)
+    | EAnn       (SomeUni f)     (Expr v f)
```

Comment:
We'll eventually need general annotations (e.g. to store source code positions). `ann` is the abbreviation that we use on the Plutus side for such annotations. Can we maybe call this `ETypeAnn` or something like that?